src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/Functions/ProcessNextDataSetVersionFunctionTests.cs

```diff
@@ -111,33 +112,48 @@ private async Task ProcessNextDataSetVersion(TaskOrchestrationContext orchestrat
         }
     }
 
-    public class CreateMappingsProcessingTests(
+    public abstract class CreateMappingsTests(
         ProcessorFunctionsIntegrationTestFixture fixture)
-        : ProcessInitialDataSetVersionFunctionTests(fixture)
+        : ProcessNextDataSetVersionFunctionTests(fixture)
     {
-        private const DataSetVersionImportStage Stage = DataSetVersionImportStage.ImportingMetadata;
+        protected const DataSetVersionImportStage Stage = DataSetVersionImportStage.CreatingMappings;
+
+        protected async Task CreateMappings(Guid instanceId)
+        {
+            var function = GetRequiredService<ProcessNextDataSetVersionFunction>();
+            await function.CreateMappings(instanceId, CancellationToken.None);
+        }
+    }
 
+    public abstract class CreateMappingMiscTests(
```

Comment:
can remove the `abstract` here

Reply:
Also, wonder if we should stick the three test classes that extend `CreateMappingsTests` inside of `CreateMappingsTests`? Just so that the nesting makes it a bit easier to digest?

Same with the three test classes that extend `ApplyAutoMappingsTests`?

Reply:
Update this one thanks.  The remaining abstract ones in this class are all base classes for the test suites rather than test suites.

Reply:
Sorry, for the second bit I was suggesting nesting the test classes inside of their abstract parent base class, just to make the discovery a little better? It will indent everything by 1 tab of course. Don't feel super strongly about it though